apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.ts

```diff
@@ -117,7 +123,15 @@ async function getEntitiesFromParams(params: AddFeatureGrantModalParams) {
     course_instance = undefined;
   }
 
-  return { institutions, institution, courses, course, course_instances, course_instance };
+  const user = await run(async () => {
+    if (!params.user_uid) return null;
+
+    const user = await selectOptionalUserByUid(params.user_uid);
+    if (!user) throw new error.HttpStatusError(400, `User not found: ${params.user_uid}`);
+    return user;
+  });
```

Comment:
Nit: while I appreciate the functional-programming style, in cases like this I feel like the traditional imperative style is more idiomatic in JS.
```suggestion
  var user : User | null = null;
  if (params.user_id) {
    user = await selectOptionalUserByUid(params.user_uid);
    if (!user) throw new error.HttpStatusError(400, `User not found: ${params.user_uid}`);
  }
```

Reply:
Fair enough 😂 done in e3f672b.